package.json

```diff
@@ -129,6 +129,8 @@
 		"page": "1.7.1",
 		"photon": "2.0.1",
 		"plugin-error": "1.0.1",
+		"preact": "8.4.2",
+		"preact-portal": "1.1.3",
```

Comment:
We should be really deliberate about using `preact` instead of `React` here. I'm not sure what it'd mean for JSX transpilation, and if it's worth the kilobytes saved. There's also quite some risk in interoperability with `@wordpress/element`. This isn't a decision we should take lightly; is there a P2 thread where we've discussed this and come to a conclusion? (I think it's conceivable that we end up using React on the frontend at some point, as more and more Gutenberg blocks require some level of interactivity on the frontend.)

Reply:
Yes, here https://jetpackcrew.wordpress.com/2019/05/16/react-for-front-end-code/

The end result was using preact produced a bundle an order of magnitude smaller that react, and as this is a front end widget potentially loaded for every visitor, it was worth it.

Personally, I'd like to use react because we'd be able to reusable the `@wordpress/*` packages, but the conclusion was bundle size mattered more.

Reply:
Ah right, I vaguely recall that discussion, thanks for referencing :sweat_smile: 

Yeah, sigh. I wonder how much longer we'll limbo around using React on the frontend if we even introduce `preact` as a drop-in. :slightly_frowning_face: